lib/timberline/config.rb

```diff
@@ -1,11 +1,13 @@
+require 'cgi'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.